src/ORM/Queries/SQLConditionalExpression.php

```diff
@@ -78,7 +78,18 @@ public function addFrom($from)
         if (is_array($from)) {
             $this->from = array_merge($this->from, $from);
         } elseif (!empty($from)) {
-            $this->from[str_replace(['"','`'], '', $from)] = $from;
+            // Make sure we don't have any pointless blank space in our From clause
+            $from = trim($from);
```

Comment:
This wasn't being done before and doesn't seem related to the bug you're fixing. What's this change for?

Reply:
I didn't want to trigger the logic below if someone made a call like `$query->addFrom(' MyTableName ')`;

I guess I could chande the regex to something like `^\s*[^\s]+\s*$`. That would match:
- an optional series of spaces followed by
- an non-optional series of none-space followed by 
- an optional series of spaces followed by

Reply:
Personally I think if people are putting whitespace around their table name, they should expect errors. But I'm not fussed either way really.

Reply:
If it's meant to imitate SQL, you can have as many spaces around your table name as you want. But, I agree it feels a bit yucky allowing spaces there.